src/light-signup.scss

```diff
@@ -61,25 +60,27 @@ $next-sass-setup-is-silent: false !default;
   margin-bottom: 5px;
 }
 
-.n-light-signup__no-spam {
+.o-email-only-signup__no-spam {
   @include oTypographySans(s);
   margin: 0;
   display: inline-block;
 }
 
-.n-light-signup__close {
-  @include nextIcon(
-	$icon-name: cross_disk,
-	$size: 20,
-	$color: #4A4A4A
+.o-email-only-signup__close {
+  @include oIconsGetIcon(
+    $icon-name: 'cross-disk',
+    $color: oColorsGetPaletteColor('grey-tint4')
```

Comment:
We usually define colors for use in components as use cases using `oColorsSetUseCase` not a huge deal but worth considering.